catalog/core/catalog-core-impl/pubsub/src/test/java/ddf/catalog/pubsub/MockSubscription.java

```diff
@@ -34,21 +34,12 @@ public MockSubscription(Subject user, DeliveryMethod dm) {
 
     /**
      * Create a Federated Subscription
-     *
-     * @param user
-     * @param compoundCriteria
-     * @param dm
+     *  @param dm
      * @param siteNames        , null if you want to query the whole Enteprise
      */
-    public MockSubscription(Subject user, DeliveryMethod dm, Set<String> siteNames) {
+    public MockSubscription(DeliveryMethod dm, Set<String> siteNames) {
         super();
         setDeliveryMethod(dm);
-        if (siteNames == null) {
```

Comment:
@vinamartin siteNames can be null, but the `if` statement exists to assign a value that is never used, so I took it out